main/src/library/BigInt.flix

```diff
@@ -95,7 +95,7 @@ namespace BigInt {
     /// A successful parse is wrapped with `OK(x)`, a parse failure is indicated by `Err(_)`.
     ///
     pub def fromString(s: String): Result[BigInt, String] = try {
-        import java.lang.String.strip();
+        import java.lang.String.strip(): String & Pure;
         import new java.math.BigInteger(String) as parseBigInt;
         Ok(s |> strip |> parseBigInt) as & Pure
```

Comment:
Is it possible to use the notation in a case like this?
I couldn't seem to figure out how, if it is even possible.